src/openzaak/components/documenten/admin.py

```diff
@@ -40,6 +41,24 @@ class GebruiksrechtenAdmin(AuditTrailAdminMixin, UUIDAdminMixin, admin.ModelAdmi
     raw_id_fields = ("informatieobject",)
     viewset = viewsets.GebruiksrechtenViewSet
 
+    def has_delete_permission(self, request, obj=None):
+        if settings.CMIS_ENABLED:
+            return False
+        else:
+            return super().has_delete_permission(request, obj)
+
+    def has_add_permission(self, request):
+        if settings.CMIS_ENABLED:
+            return False
+        else:
+            return super().has_add_permission(request)
+
+    def has_change_permission(self, request, obj=None):
+        if settings.CMIS_ENABLED:
+            return False
+        else:
+            return super().has_change_permission(request, obj)
+
```

Comment:
Could you move this to a mixin class? This prevents the same code in `EnkelvoudigInformatieObjectAdmin` and `ObjectInformatieObjectAdmin`

Reply:
Done!